docs/usage.rst

```diff
@@ -55,6 +55,18 @@ Note that you can still implicitly provide a schema in SQL statements, e.g.:
     used.
 
 
+Timeout
+=======
+
+Crate-PDO uses HTTP requests to query CrateDB. If a query lasts longer than the specified
```

Comment:
... uses the CrateDB HTTP protocol (link to documentation). Setting the PDO attribute `PDO::ATTR_TIMEOUT` will cause the HTTP request to cancel and raise a timeout exception (? please test what really happens) after the specified timeout. Cancelling the HTTP connection, however, does not cancel/kill the executed statement on the server.